Tarea 1/tarea-1-index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Monthly income calculator</title>
+    <link rel="stylesheet" href="file:///C:/Users/Seba/Documents/GitHub/5384/clase-5/Tarea%201/tarea-1.css">
```

Comment:
I think this is not necessary. Also, this will not work because it's a URL of your system.

Reply:
Fixed by deleting the path towards my users C: HD.